Includes response output: spo app until spo eventreceiver commands. Closes #3929 (#5540)
pnp/cli-microsoft365: docs/docs/cmd/spo/eventreceiver/eventreceiver-get.mdx

## Examples

> Retrieve event receiver in the specified web with given name.

Reviewer: ```suggestion
Retrieve an event receiver in the specified web with the given name.
```

> Retrieve an event receiver in the specified web with the given name.

Reviewer: ```suggestion
Retrieve an event receiver with the given name in the specified site.
```

```sh
m365 spo eventreceiver get --webUrl https://contoso.sharepoint.com/sites/contoso-sales --name 'PnP Test Receiver'
```

> Retrieve event receiver in the specified site with given id.

Reviewer: ```suggestion
Retrieve an event receiver in the specified site with the given id.
```

There are more occurrences of this in the current file

> Retrieve an event receiver in the specified site with the given id.

Reviewer: ```suggestion
Retrieve an event receiver with the given id in the specified site.
```

```sh
m365 spo eventreceiver get --webUrl https://contoso.sharepoint.com/sites/contoso-sales --scope site --id c5a6444a-9c7f-4a0d-9e29-fc6fe30e34ec
```

> Retrieve an event receiver for list with given title in the specified web with the given name.

Reviewer: ```suggestion
Retrieve an event receiver with the given name for a list with the given title in the specified site.
```

```sh
m365 spo eventreceiver get --webUrl https://contoso.sharepoint.com/sites/contoso-sales --listTitle Events --name 'PnP Test Receiver'
```

> Retrieve an event receiver for list with given ID in the specified web with the given id.

Reviewer: ```suggestion
Retrieve an event receiver with the given id for a list with the given ID in the specified site.
```

```sh
m365 spo eventreceiver get --webUrl https://contoso.sharepoint.com/sites/contoso-sales --listId '202b8199-b9de-43fd-9737-7f213f51c991' --id c5a6444a-9c7f-4a0d-9e29-fc6fe30e34ec
```

> Retrieve an event receiver for list with given url in the specified web with the given name.

Reviewer: ```suggestion
Retrieve an event receiver with the given name for a list with the given url in the specified site.
```
